test/presenters/bill-runs/two-part-tariff/review-bill-run.presenter.test.js

```diff
@@ -12,13 +12,20 @@ const ReviewBillRunPresenter = require('../../../../app/presenters/bill-runs/two
 
 describe('Review Bill Run presenter', () => {
   describe('when there is data to be presented for review', () => {
+    const filterIssues = undefined
```

Comment:
Maybe wrap these in another describe block like 'and no filter has been applied'? That way you can declare all the filter variables inside that describe block and see that they are undefined

Reply:
Yes that would be better.